README.md

```diff
@@ -1,15 +1,73 @@
 
+### **TabNine Deep Cloud:**
+___
+TabNine requires consumption of memory resources when being run locally on your computer. It may incur some latency that is not optimal to all PC’s. With that in mind, TabNine has developed a Cloud solution, called TabNine Deep Cloud.
+
+
+We understand that users concerned with their privacy prefer to keep their code on their own machine. Rest assured that we’re taking the following steps to address this concern:
+For individual developers, we are working on a reduced-size model which can run on a laptop with reasonable latency. Update: we’ve released TabNine Local.
+For enterprise users, we will soon roll-out the option to license the model and run it on your own hardware. We can also train a custom model for you which understands the unique patterns and style within your codebase. If this could be relevant to you or your team, we would love to hear more about your use case at enterprise@tabnine.com.
+Enabling TabNine Deep Cloud sends small parts of your code to our servers to provide GPU-accelerated completions.
+Other than for the purpose of fulfilling your query, your data isn’t used, saved or logged in any way.
+
+**[You can enable TabNine Deep Cloud for free by signing up TabNine Professional](https://www.tabnine.com/trial/)**
+
+**[You can read more about Deep Completion](https://www.tabnine.com/blog/deep)**
+
+### auto_complete: false
+___
 The TabNine sublime plugin disables sublime's [built in autocomplete](https://www.sublimetext.com/docs/3/auto_complete.html).  
 It does that, because sublime's builtin autocomplete does not support all the features required by TabTine.  
 If you ever uninstall TabNine, remember to re-enable it.
+
+### **Frequently Asked Questions:**
+___
+<details><summary>Is TabNine simple to use?</summary>
+<p>TabNine works for all programming languages.
+TabNine does not require any configuration in order to work.
+TabNine does not require any external software (though it can integrate with it).
+Since TabNine does not parse your code, it will never stop working because of a mismatched bracket.
+</p>
+</details>
+
+<details><summary>Will TabNine leak my source code?</summary>
+<p>By default, TabNine makes web requests only for the purposes of downloading updates and validating registration keys. In this case your code is not sent anywhere, even to TabNine servers.
+You may opt in to TabNine Deep Cloud, which allows you to use TabNine’s servers for GPU-accelerated completions powered by a deep learning model. If sending code to a cloud service is not possible, we also offer a self-hosted option. 
+Contact us at enterprise@tabnine.com.
+</p>
+</details>
+
+### **Note:**
+___
+A note on licensing: this repo includes source code as well as packaged TabNine binaries. The MIT license only applies to the source code, not the binaries. The binaries are covered by the [TabNine End User License Agreement](https://tabnine.com/eula).
```

Comment:
we need to change this, the link should be to the new terms of use page

Reply:
Will be fixed in the future versions, that will include different messaging and SEO optimizations